kpi/models/object_permission.py

```diff
@@ -185,8 +185,9 @@ def get_cached_code_names(model_: models.Model = None) -> dict:
 
 
 @cache_for_request
-def get_perm_ids_from_code_names(code_names: Union[str, list],
-                                 model_: models.Model = None) -> Union[int, list]:
+def get_perm_ids_from_code_names(
+    code_names: Union[str, list, tuple, set], model_: models.Model = None
+) -> Union[int, list]:
```

Comment:
Can we use `Collection`? https://stackoverflow.com/a/61155469/2402324

Reply:
OK, no:
```
In [1]: from collections.abc import Collection                                                                                       

In [2]: _list = list('hi there')                                                                                                     

In [3]: _tup = tuple(_list)                                                                                                          

In [4]: _set = set(_list)                                                                                                            

In [5]: isinstance(_list, Collection)                                                                                                
Out[5]: True

In [6]: isinstance(_tup, Collection)                                                                                                 
Out[6]: True

In [7]: isinstance(_set, Collection)                                                                                                 
Out[7]: True

In [8]: isinstance('ouch', Collection)                                                                                               
Out[8]: True
```

Reply:
I tried:. Also tried (Iterable, Sequence), no luck.